### PR TITLE
Expire vertica passwords job DSL

### PR DESCRIPTION
**Pull request by HassanJaveed84**
analytics-secure PR: https://github.com/edx-ops/analytics-secure/pull/311

### CODE REVIEW

File: dataeng/jobs/analytics/ExpireVerticaPassword.groovy [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                        credentials('1')
+                    }
+                    extensions {
+                        relativeTargetDirectory('analytics-tools')
+                        pruneBranches()
+                        cleanAfterCheckout()
+                    }
+                }
+            }
+            //triggers common_triggers(allVars)
```
Comment (HassanJaveed84) [inline dataeng/jobs/analytics/ExpireVerticaPassword.groovy]: Will uncomment and add trigger before merge.

File: dataeng/jobs/analytics/ExpireVerticaPassword.groovy [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            publishers {
+                extendedEmail {
+                    recipientList('$NOTIFY_LIST')
+                    defaultSubject('You Vertica password has expired')
+                    defaultContent('This is to inform you that you Vertica password has expired. ' +
+                    'Please use vsql to set a new password.')
+                    triggers {
+                        success {
+                            sendTo {
+                                recipientList()
```
Comment (pwnage101) [inline dataeng/jobs/analytics/ExpireVerticaPassword.groovy]: Is there a way to make this send via Bcc?

File: dataeng/resources/expire-vertica-password.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+set -ex
+
+# Setup
+cd $WORKSPACE/analytics-tools/vertica
+pip install -r requirements.txt
+
+python expire_user_passwords.py \
+    -c $CREDENTIALS \
+    -e $EXCLUDE \
+    -m $MAPPING
```
Comment (pwnage101) [inline dataeng/resources/expire-vertica-password.sh]: nit: preference to use long form of arguments, e.g. --credentials.

File: dataeng/jobs/analytics/ExpireVerticaPassword.groovy (hunk 2 above)
Comment (HassanJaveed84) [reply dataeng/jobs/analytics/ExpireVerticaPassword.groovy]: Yes, by prepending bcc: to the email, that is, bcc:someone@example.com. I'll make the change in the script.

File: dataeng/resources/expire-vertica-password.sh (hunk 3 above)
Comment (HassanJaveed84) [reply dataeng/resources/expire-vertica-password.sh]: Done.

File: dataeng/jobs/createJobs.groovy [hunk 4]
```diff
@@ -126,7 +127,8 @@ def taskMap = [
     LOAD_PAYPAL_CASEREPORT_TO_VERTICA_JOB: PayPalCaseReportLoadJob,
     WAREHOUSE_TRANSFORMS_JOB: WarehouseTransformsJob,
     DBT_SOURCE_FRESHNESS_JOB: DBTSourceFreshnessJob,
-    SNOWFLAKE_REFRESH_SNOWPIPE_JOB: SnowflakeRefreshSnowpipeJob,
+    SNOWFLAKE_REFRESH_SNOWPIPE_JOB: SnowflakeRefreshSnowpipeJob,*/
```
Comment (bmedx) [inline dataeng/jobs/createJobs.groovy]: Assume you'll uncomment this before merging too?

**Pull request merged**